jmxtrans-core/pom.xml

```diff
@@ -132,4 +139,40 @@
 		</plugins>
 	</build>
 
+	<profiles>
+		<profile>
+			<id>windows_profile</id>
+			<activation>
+				<os>
+					<family>windows</family>
+				</os>
+			</activation>
+			<properties>
+				<toolsjar>${java.home}/../lib/tools.jar</toolsjar>
```

Comment:
It seems that at the moment, all those different profiles set the same value to `toolsjar`. So we can inline this property for the moment, and introduce profiles once we know about an actual difference between OS.